src/components/Footer.js

```diff
@@ -1,50 +1,123 @@
+                                    >
+                                        {member.name}
+                                    </GoldHoverLink>
+                                </li>
+                            ))}
+                        </ul>
+                    </div>
+                </Col>
+                <Col md="auto">
+                    <div className="p-4">
+                        <h6 className="text-uppercase mb-3 font-weight-bold">Follow us</h6>
+
+                        {/* <!-- Facebook --> */}
+                        <a
+                            className="btn btn-primary btn-floating me-1 rounded-circle border-0"
+                            style={{ backgroundColor: "#3b5998", padding: "0.5rem 0.95rem" }}
+                            href="#!"
+                            role="button"
+                        ><i className="fa fa-facebook-f"></i></a>
+
+                        {/* <!-- Twitter --> */}
+                        <a
+                            className="btn btn-primary btn-floating m-1 border-0 rounded-circle"
+                            style={{ backgroundColor: "#55acee", padding: "0.5rem 0.8rem" }}
+                            href="#!"
+                            role="button"
+                        ><i className="fa fa-twitter"></i></a>
+
+                        {/* <!-- Google --> */}
+                        <a
+                            className="btn btn-primary btn-floating m-1 border-0 rounded-circle"
+                            style={{ backgroundColor: "#dd4b39", padding: "0.5rem 0.85rem" }}
+                            href="#!"
+                            role="button"
+                        ><i className="fa fa-google"></i></a>
+
+                        {/* <!-- Instagram --> */}
+                        <a
+                            className="btn btn-primary btn-floating m-1 border-0 rounded-circle"
+                            style={{ backgroundColor: "#ac2bac", padding: "0.5rem 0.85rem" }}
+                            href="#!"
+                            role="button"
+                        ><i className="fa fa-instagram"></i></a>
+
+                        {/* <!-- Github --> */}
+                        <a
+                            className="btn btn-primary btn-floating m-1 border-0 rounded-circle"
+                            style={{ backgroundColor: "#333333", padding: "0.5rem 0.85rem" }}
+                            href="https://github.com/Yashdon999/Resume-Generator"
+                            role="button"
+                        ><i className="fa fa-github"></i></a>
+
+                    </div>
+                </Col>
+            </Row>
+            <Row>
+                <div
+                    class="text-center p-3"
+                    style={{ backgroundColor: "rgba(0, 0, 0, 0.2)" }}
+                >
+                    Copyright © 2023 Resume Generator
+                </div>

```

Comment:
you can use 

```javascript
{new Date().getFullYear()}
```

Instead of explicitly hard coding the year

Reply:
no problem